### PR TITLE
Added secondary route and modal

### PR DESCRIPTION
**Pull request by Kavil-Jain**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -6,9 +6,11 @@ import {PageNotFoundComponent} from "./user/page-not-found/page-not-found.compon
 import {WelcomeComponent} from "./user/welcome/welcome.component";
 import {AuthGuard} from "./guard/auth.guard";
 import {DetailContainerComponent} from "./hero/detail/detail-container/detail-container.component";
+import {LoginComponent} from "./user/login/login.component";
 
 const routes: Routes = [
   {path: "welcome", component: WelcomeComponent},
+  {path: "login", component: LoginComponent, outlet: "loginPage"},
```
Comment (rachanakp-inf) [inline src/app/app-routing.module.ts]: can you rename it to modal or popup so that this outlet can be used by all components that could be / would be modals

File: src/app/app.component.scss [hunk 2]
```diff
@@ -1,4 +0,0 @@
-img{
-  width: 100%;
```
Comment (rachanakp-inf) [inline src/app/app.component.scss:2]: css not required?

File: src/app/shared/shared.module.ts [hunk 3]
```diff
@@ -29,7 +29,6 @@ import { LoaderComponent } from './loader/loader.component';
     FooterComponent,
     DetailDataComponent,
     LoaderComponent,
-    DetailDataComponent,
```
Comment (rachanakp-inf) [inline src/app/shared/shared.module.ts:32]: why was this removed?

File: src/app/user/login/login.component.ts [hunk 4]
```diff
@@ -70,4 +70,9 @@ export class LoginComponent implements OnInit {
       this.displayPasswordMessage = Object.keys(control.errors).map(key => (<any>this.validationMessages.password)[key]).join(' ');
     }
   }
+
+  closeModal(){
+    this._router.navigate([{outlets: {loginPage: null, primary: ['welcome']}}]);
+    document.getElementById('form-group')!.style.display='none';
```
Comment (rachanakp-inf) [inline src/app/user/login/login.component.ts]: why display none for style? the component should go once you leave the route

File: src/app/shared/shared.module.ts (hunk 3 above)
Comment (Kavil-Jain) [reply src/app/shared/shared.module.ts:32]: DetailDataComponent was duplicated..

File: src/app/app.component.scss (hunk 2 above)
Comment (Kavil-Jain) [reply src/app/app.component.scss:2]: yes not required.. removed background image
